pandas/tests/extension/test_format.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
```

Comment:
Dunno if there's a better spot for this test. It's getting at the case I mentioned in https://github.com/pandas-dev/pandas/issues/26837#issuecomment-967268492, where printing the repr called asarray on the EA (and asarray on all of its elements too)

Reply:
could setup another class in the EA array tests themselves i think is better

Reply:
`extension/test_common.py` already has a DummyArray that I can reuse. I'll put it there.